cmd/rbe_configs_upload/rbe_configs_upload.go

```diff
@@ -91,7 +94,7 @@ func newStorage(ctx context.Context) (*storageClient, error) {
 	}
 	return &storageClient{
 		client:     c,
-		bucketName: "rbe-bazel-toolchains",
+		bucketName: "rbe-toolchain",
```

Comment:
Will clients be looking in an old bucket?  If so, maybe delete the contents once we're rolling on the new one, so any breakages are immediate/intentional?

Reply:
TL; DR no special action necessary here

No one should be using the old bucket yet because usage instructions haven't been communicated yet. I had to rename the bucket because internally the new bucket is hosted in a different GCP project and it's not trivial to move a bucket to a different project ID (can require waiting 90 days I think before a bucket ID name is available after deletion).